web/documentserver-example/nodejs/app.js

```diff
@@ -332,9 +332,8 @@ app.post('/convert', (req, res) => { // define a handler for converting files
   const lang = req.body.lang ? req.body.lang : null;
   const fileUri = req.DocManager.getDownloadUrl(fileName, true);
   const fileExt = fileUtility.getFileExtension(fileName, true);
-  const internalFileExt = 'ooxml';
-  let convExt = req.body.fileExt ? req.body.fileExt : internalFileExt;
-  if (req.body.forceConv) convExt = req.body.forceConv;
+  const conversionExtension = req.body.fileExt ? req.body.fileExt : 'ooxml';
+  const { keepOriginal } = req.body;
```

Comment:
Now it looks just like renaming variables... However, return `internalFileExt`, it is needed to know what is `ooxml`